server/src/fakedb_writer.js

```diff
@@ -0,0 +1,62 @@
+const csv = require('csv-parser');
```

Comment:
Replace the file name to something like "data_importer.js", so we can use this file for other purposes (e.g. writing an indexing pipeline that crawls data sources and imports into db).